hud-data/ahud-cc.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "ahud-cc",
+  "author": "Jakadak",
+  "contributors": [
+    "Hikyle",
+    "Hypnootize",
+    "Peaches",
+    "SiFWolf",
+    "Quik",
+    "Griever"
+  ],
+  "releaseDate": "2024-07-25",
+  "parent": "ahud",
+  "resources": [
+    "https://i.imgur.com/dsAQAeM.png",
+    "https://i.imgur.com/jnLvG82.png",
+    "https://i.imgur.com/acjZPmi.png",
+    "https://i.imgur.com/6uG5LUw.png",
+    "https://i.imgur.com/9UJR0gP.jpg",
+    "https://i.imgur.com/wicJuVg.jpg",
+    "https://i.imgur.com/z1TlYCe.png",
+    "https://i.imgur.com/I4QkmZG.jpg",
+    "https://i.imgur.com/yasPpKx.png",
+    "https://i.imgur.com/yWW5unT.jpg",
+    "https://i.imgur.com/vS86iQv.jpg",
+    "https://i.imgur.com/2Alfd01.jpg",
+    "https://i.imgur.com/TJmP2wd.jpg",
+    "https://i.imgur.com/Bnmc2Cz.png",
+    "https://i.imgur.com/EfQG3FE.png",
+    "https://i.imgur.com/yj3ny7D.jpg",
+    "https://i.imgur.com/BZB8QyN.jpg",
+    "https://i.imgur.com/92mh8vI.jpg",
+    "https://i.imgur.com/ihvvGb3.jpg",
+    "https://i.imgur.com/dC41F3U.png",
+    "https://i.imgur.com/GtOYXLe.jpg",
+    "https://i.imgur.com/rN5xzbI.jpg",
+    "https://i.imgur.com/uxzGC9x.png"
```

Comment:
HUD screenshots are typically put into `hud-resources/<your_hud>` and then their names (without extension) are referenced here in `"resources"`.

Using links here instead may work, but it'd be something I'd verify with @mastercoms first since I'm not sure

Reply:
links are not allowed, besides youtu.be links